youtube_dl/extractor/hrti.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    sanitized_Request,
+    ExtractorError
+)
+
+
+class HRTiIE(InfoExtractor):
+    # from app.min.js
+    APP_VERSION = "1.1"
+    APP_PUBLICATION_ID = "all_in_one"
+
+    _VALID_URL = r'https?://hrti.hrt.hr/#/video/show/(?P<id>[0-9]+)/(?P<name>(\w|-)+)?'
+    _TEST = {
+        'url': 'https://hrti.hrt.hr/#/video/show/2181385/republika-dokumentarna-serija-16-hd',
+        'info_dict': {
+            'id': '2181385',
+            'ext': 'mp4',
+            'name': 'REPUBLIKA, dokumentarna serija (4_6)-2251938',
+        },
+        'skip': 'Requires login'
+    }
+
+    def _login(self):
+        (username, password) = self._get_login_info()
+
+        if username is None or password is None:
+            raise ExtractorError(
+                'Unable to login, incorrect username and/or password', expected=True)
```

Comment:
Updated the code to reflect the fact **credentials** are required. Reporting on invalid username/password is one on an actual login attempt.